runtime/JavaScript/src/antlr4/CharStream.d.ts

```diff
@@ -1,5 +1,9 @@
-export declare class CharStream {
+/* Copyright (c) 2012-2022 The ANTLR Project. All rights reserved.
```

Comment:
it's 2023 :-)

Reply:
My last information about copyrights in ANTLR4 is that we always keep the original ones and don't update any year, but maybe that only belongs to the first year, not the second one.

Reply:
Once I have a definite answer from @parrt I'll update them.